packages/shared/src/atoms/ThemeSwitch/ThemeSwitch.style.tsx

```diff
@@ -13,6 +13,7 @@ export const ButtonThemeSwitch = styled(IconButton)(({ theme }) => ({
     theme.palette.mode === 'dark'
```

Comment:
not in your changes, but could be another possible place to implement it

`getDefaultThemeBackgroundColor `
https://github.com/lifinance/jumper.exchange/pull/502/files#r1157138142